addons/mrp/static/src/js/mrp_bom_report.js

```diff
@@ -242,6 +249,30 @@ var MrpBomReport = stock_report_generic.extend({
             $parent.remove();
         });
     },
+    _unfoldedIDs: function() {
+        return _.map(this.$el.find('.o_mrp_bom_foldable').closest('tr'), function (el) {
+            return $(el).data('id');
+        });
```

Comment:
```suggestion
        return _.map(this.$('.o_mrp_bom_foldable').closest('tr'), el => $(el).data('id'));
```
Use `$` instead of `$el.find` (shorter and helper made for this purpose) and you can make the code shorter by using an arrow function (in which case it will automatically returns the statement if there is no braces, so no need to use `return` if there is only one line statement).